ludwig/data/dataframe/dask.py

```diff
@@ -155,12 +156,22 @@ def map_partitions(self, series, map_fn, meta=None):
         meta = meta if meta is not None else ("data", "object")
         return series.map_partitions(map_fn, meta=meta)
 
-    def map_batches(self, series, map_fn):
+    def map_batches(self, series, map_fn, enable_tensor_extension_casting=True):
+        """Map a function over batches of a Dask Series.
+
+        Args:
+            series: Dask Series
+            map_fn: Function to apply to each batch
+            enable_tensor_extension_casting: Whether to enable tensor extension casting at the end of the Ray Datasets
+                map_batches call. This is useful in cases where the output is not supported by the ray Tensor dtype
```

Comment:
nit: "Should be disabled when the output tensors are not supported by the ray Tensor dtype extension, such as when the output consists of ragged tensors."

Or did I get that mixed up?

Reply:
That's right! Will add that as a comment.